packages/elements-dev-portal/src/components/UpgradeToStarter/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { Box, Flex, Icon } from '@stoplight/mosaic';
+import React from 'react';
+
+export const UpgradeToStarter = () => (
+  <Flex
+    as="a"
+    href="https://stoplight.io/pricing/"
```

Comment:
Would you maybe consider making this a `target="_blank" rel="noreferrer noopener"`? @philsturgeon ?